src/hera/_cli/__init__.py

```diff
@@ -0,0 +1 @@
+"""Hera's command line interface."""
```

Comment:
I was just copying the base package @elliotgunton used in the original closed PR. At least in the short term, it would make sense to me to define it as a private package at least until it's more set in stone. I'm otherwise very ambivalent about it though.